packages/schematics/package.json

```diff
@@ -1,21 +1,42 @@
 {
   "name": "@nrwl/schematics",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Nrwl Extensions for Angular: Schematics",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "RxJS",
+    "Angular",
+    "Workspace",
+    "NgRx",
+    "Schematics",
+    "Angular CLI"
+  ],
   "bin": {
     "create-nx-workspace": "./bin/create-nx-workspace.js",
     "nx": "./src/command-line/nx.js"
   },
   "main": "index.js",
   "types": "index.d.js",
-  "peerDependencies": {},
+  "peerDependencies": { },
   "author": "Victor Savkin",
   "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "homepage": "https://github.com/nrwl/nx#readme",
   "schematics": "./src/collection.json",
   "dependencies": {
+    "@ngrx/schematics": "^5.2.0",
```

Comment:
we should replace 5.2.0 with a placeholder, provision the version in lib-versions, and replace it when creating a package.

Reply:
I will make a separate issue after this PR.